.github/CODEOWNERS

```diff
@@ -15,9 +37,10 @@ docs/bigquery_datatransfer/ @googleapis/api-bigquery
 bigquery_storage/           @googleapis/api-bigquery
 docs/bigquery_storage/      @googleapis/api-bigquery
 
-# Danny Hermes and Tres Seaver co-authored the BigTable library.
-bigtable/                   @tseaver
-docs/bigtable/              @tseaver
+# Pubsub
+pubsub/                     @anguillanneuf @plamut
+docs/pubsub                 @anguillanneuf @plamut
+
 
 # Bill Prin is the primary maintainer of the Stackdriver libraries.
```

Comment:
This person seems to be gone, is it OK to remove this section? @tseaver

Reply:
I think this is safe to remove. This will default to you.